google-cloud-bigquery/CHANGELOG.md

```diff
@@ -1,5 +1,16 @@
 # Release History
 
+### 1.14.0 / 2019-08-20
+
+#### Features
+
+* Update documentation
```

Comment:
I would list this last. This isn't more important than overriding the service endpoint.

Reply:
In fact, the only documentation changes I see are related to the service endpoint.

Reply:
So this may be incorrect, as I don't know which documentation has been updated, exactly.